content/en/developers/marketplace.md

```diff
@@ -108,36 +108,43 @@ Technology Partners can request access to the private Marketplace repository by
 
 The Datadog Development Toolkit command (`ddev`) allows you to create scaffolding when you are first developing your integration, by spinning up a skeleton of all the assets and metadata for your tile.
 
-Ensure that Python 3.8 or higher is installed.	
+Ensure that Python 3.8 or higher is installed (https://www.python.org/downloads/).	
 
-The latest released version may be installed from [PyPI][23]:
+To avoid a range of potential environment conflicts, we recommend going into the directory where you cloned the repository and creating a simple virtual environment with the following command: `python3 -m pip install virtualenv --user`
 
-`python -m pip install --upgrade "datadog-checks-dev[cli]`
+The latest released version of the Datadog Development Toolkit may then be installed from [PyPI][23]:
+
+`python -m pip install --upgrade "datadog-checks-dev[cli]"`
+
+**Note:** if you are using Z shell, you may need to use escape characters: `python -m pip install --upgrade datadog-checks-dev\[cli\]` 
 
 Set the location of to the cloned repository:
 
 #### Marketplace:
 
-`ddev config set marketplace /path/to/marketplace`
+`ddev config set marketplace /path/to/marketplace_directory` (For example: /Users/yourname/Documents/marketplace)
```

Comment:
Hi @kayayarai, the path I gave is just an example. The partner may have a very different path, so I think it would be confusing to tell them use /Users/yourname/Documents/marketplace directly in the command. 

If we shouldn't include an example, or it's too verbose, I will make the small change so the path reads "path/to/marketpace_directory.

Reply:
Also, I believe the single ticks were recommended by the Docs team in the original PR, and they seem to be displaying correctly on the live page: https://docs.datadoghq.com/developers/marketplace/. Are they necessary?

Reply:
Added the backticks and removed the example language.